sale_elaboration/models/sale_order.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+def _execute_onchanges(records, field_name):
```

Comment:
I know you don't like onchange_helper, but you could avoid code duplicates like this...